crates/core_simd/tests/helpers/biteq.rs

```diff
@@ -0,0 +1,127 @@
+pub(crate) trait BitEq {
+    fn biteq(&self, other: &Self) -> bool;
+    fn fmt(&self, f: &mut core::fmt::Formatter) -> core::fmt::Result;
+}
+
+macro_rules! impl_biteq {
+    { integer impl BitEq for $($type:ty,)* } => {
+        $(
+        impl BitEq for $type {
+            fn biteq(&self, other: &Self) -> bool {
```

Comment:
```suggestion
            fn bit_eq(&self, other: &Self) -> bool {
```

Reply:
That would actually run against the convention for the bit ops, where and, or, and xor were defined with methods bitand, bitor, and bitxor.

I mean it could have the underscore anyway.

Reply:
Oh, I didn't actually notice that the bit ops were weird like that. I retract my suggestion then!